tests/unit/mixins/createFormLabelMixin.ts

```diff
@@ -0,0 +1,109 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { VNode } from '@dojo/interfaces/vdom';
+import createWidgetBase from '../../../src/createWidgetBase';
+import createFormLabelMixin from '../../../src/mixins/createFormLabelMixin';
+
+const formLabelWidget = createWidgetBase
+	.mixin(createFormLabelMixin);
+
+registerSuite({
+	name: 'mixins/createFormLabelMixin',
+	construction() {
+		const formLabelMixin = createFormLabelMixin();
+
+		assert.isDefined(formLabelMixin);
+	},
+	getNode() {
+		const formfield = formLabelWidget.override({
+			classes: ['foo', 'bar']
+		})();
+		formfield.setProperties({
+			label: 'baz'
+		});
+		const vnode = <VNode> formfield.__render__();
+
+		assert.strictEqual(vnode.vnodeSelector, 'label.foo.bar');
+	},
+	getFormFieldNodeAttributes() {
+		const formfield = formLabelWidget({
+			tagName: 'input',
+			properties: {
+				value: 'foo',
+				maxlength: 100,
+				randomProp: 'qux'
+			}
+		});
+
+		let vnode = <VNode> formfield.__render__();
+		let inputfield = vnode.children![0];
+
+		assert.strictEqual(inputfield.vnodeSelector, 'input');
+		assert.strictEqual(inputfield.properties!['value'], 'foo');
+		assert.strictEqual(inputfield.properties!['maxlength'], '100');
+		assert.isUndefined(inputfield.properties!['randomProp']);
```

Comment:
should it be randomprop for consistency, or should property names be camelCase except when they reflect an html attribute name?